nautobot/docs/user-guide/feature-guides/getting-started/interfaces.md

```diff
@@ -47,12 +48,13 @@ You will now see the `Interface Template` form:
 1. Add the `ae0` Interface Template
     * `Manufacturer` will auto-populate to the Manufacturer of the Device Type you are editing
     * `Device Type` will auto-populate to the Device Type you are editing
-    * Populate a `Name`
+    * Populate a `Name` of `ae0`
     * Select a `Type` of `Link Aggregation Group (LAG)` from the drop-down selector
     * Add a `Description` and `Label` (optional)
 2. Click `Create and Add More`
-3. Create the `xe-` Interfaces
-    * This example shows bulk creation using a range (`xe-[0-1]/0/[0-9]`) in `Name`
+3. Create the `xe-` Interfaces Template
+    * Keep the same `Manufacturer` and `Device Type` as the previous step
+    * Populate a `Name` of `xe-[0-1]/0/[0-9]`. Using the syntax of regular expressions for numbers in the name field like this will cause a bulk creation using the range of numbers provided
```

Comment:
These aren't actually regular expressions in the general sense, just the specific syntax `[<start>-<end>]` is specially supported.

Reply:
Gotcha, I will update the phrasing then to reflect that important detail.